svthalia/concrexit
# Add pagination for liked photos

Closes #2668 



### Summary
Added pagination for the liked photos page.

### How to test
Steps to test the changes you made:
1. Go to liked photos
2. Observe that there is pagination
3. See that the pagination works as intended (e.g. with ordering of pictures).

Should the picture order be changed, e.g. to be in order of 'when liked'? Currently it seems to return in the order of the pictures inside each album. Should I add this to the `order_by` in some way, either to keep the current behavior or get something else?
